test/plugin/test_parser_apache_error.rb

```diff
@@ -1,11 +1,11 @@
 require_relative '../helper'
 require 'fluent/test/driver/parser'
-require 'fluent/plugin/parser'
+require 'fluent/plugin/parser_apache_error'
 
 class ApacheErrorParserTest < ::Test::Unit::TestCase
   def setup
     Fluent::Test.setup
-    @parser = Fluent::Test::Driver::Parser.new(Fluent::Plugin.new_parser('apache_error'))
+    @parser = Fluent::Test::Driver::Parser.new(Fluent::Plugin::ApacheErrorParser.new)
```

Comment:
This instance is not parser, but test driver instance. I know it's too late to notify about it, but I want not to leave it as-is because it's misleading for following developers.
@okkez Could you fix it?